Install-DVWA.sh

```diff
@@ -0,0 +1,182 @@
+        mysql -u "$mysql_user" -e "CREATE USER 'dvwa'@'localhost' IDENTIFIED BY 'p@ssw0rd';" &>/dev/null &&
+        mysql -u "$mysql_user" -e "GRANT ALL PRIVILEGES ON dvwa.* TO 'dvwa'@'localhost';" &>/dev/null &&
+        mysql -u "$mysql_user" -e "FLUSH PRIVILEGES;" &>/dev/null
+    fi
+
+    if [ $? -eq 0 ]; then
+        echo "$(get_language_message "\033[92mMySQL commands executed successfully.\033[0m" "\033[92mComandos MySQL ejecutados con éxito.\033[0m")"
+    else
+        echo -e "$(get_language_message "\033[91mError: Unable to execute MySQL commands. Please check your MySQL credentials." "\033[91mError: No se pueden ejecutar los comandos de MySQL. Por favor, verifique sus credenciales de MySQL.")"
+    fi
+}
+
+# ASCII Art
+echo -e "\033[96m\033[1m
+                  ██████╗ ██╗   ██╗██╗    ██╗ █████╗                    
+                  ██╔══██╗██║   ██║██║    ██║██╔══██╗                   
+                  ██║  ██║██║   ██║██║ █╗ ██║███████║                   
+                  ██║  ██║╚██╗ ██╔╝██║███╗██║██╔══██║                   
+                  ██████╔╝ ╚████╔╝ ╚███╔███╔╝██║  ██║                   
+                  ╚═════╝   ╚═══╝   ╚══╝╚══╝ ╚═╝  ╚═╝                   
+                                                                        
+  ██╗███╗   ██╗███████╗████████╗ █████╗ ██╗     ██╗     ███████╗██████╗ 
+  ██║████╗  ██║██╔════╝╚══██╔══╝██╔══██╗██║     ██║     ██╔════╝██╔══██╗
+  ██║██╔██╗ ██║███████╗   ██║   ███████║██║     ██║     █████╗  ██████╔╝
+  ██║██║╚██╗██║╚════██║   ██║   ██╔══██║██║     ██║     ██╔══╝  ██╔══██╗
+  ██║██║ ╚████║███████║   ██║   ██║  ██║███████╗███████╗███████╗██║  ██║
+  ╚═╝╚═╝  ╚═══╝╚══════╝   ╚═╝   ╚═╝  ╚═╝╚══════╝╚══════╝╚══════╝╚═╝  ╚═╝                                                                        
+\033[0m"
+
+welcome_message=$(get_language_message "\033[96mWelcome to the DVWA installer!\033[0m" "\033[96m¡Bienvenido al instalador de DVWA!\033[0m")
+echo -e "$welcome_message"
+
+echo
+# Start of the installer
+
+# Update the repositories
+update_message=$(get_language_message "\e[96mUpdating repositories...\e[0m" "\e[96mActualizando repositorios...\e[0m")
+echo -e "$update_message"
+apt update
+
+# Chek if the dependencies are installed
+dependencies_message=$(get_language_message "\e[96mVerifying and installing necessary dependencies...\e[0m" "\e[96mVerificando e instalando dependencias necesarias...\e[0m")
+echo -e "$dependencies_message"
+
+check_program apache2
+check_program mariadb-server
+check_program mariadb-client
+check_program php
+check_program php-mysql
+check_program php-gd
+check_program libapache2-mod-php
+check_program git
+
+# Download the repo
+download_message=$(get_language_message "\e[96mDownloading DVWA from GitHub...\e[0m" "\e[96mDescargando DVWA desde GitHub...\e[0m")
+echo -e "$download_message"
+git clone https://github.com/digininja/DVWA.git /var/www/html/DVWA
```

Comment:
This fails if the directory already exists. Could you check and ask a question about what they want to do if it does exist.

This may happen if they've tried to install by hand, failed, and then moved to this script.